tests/integration/database-teardown.ts

```diff
@@ -0,0 +1,12 @@
+import {executeSql} from "../../src/services/connection-pool";
+import {allTables, TableDetails} from "../../src/services/table-details";
+
+export const databaseTearDown = async () => {
+    // only run if database is not containerized - no point in tearing down data on an ephemeral environment
```

Comment:
I assume the e2e do not use that otherwise I would think when it comes to testing we might want to start from a clean slate and not preserve state across tests?

Reply:
I have removed this file after confirming with Tanio that he no longer needs it, as he has fixed this in the pipeline. This was indeed the intention with the Dockerized database, however Tanio was hesitant to work on Docker-in-Docker so we added in both this and the `USE_CONTAINERIZED_DATABASE` variable, temporarily, to help him out.